internal/lvmd/testutils/test_utils.go

```diff
@@ -77,3 +79,7 @@ func CleanLoopbackVG(name string, loops []string, files []string) error {
 
 	return nil
 }
+
+func RequireRoot(t *testing.T) {
```

Comment:
Why is it necessary to pass this require root here? Cant we just reuse the internal package directly?

Reply:
In theory, we could, but both `internal/lvmd/testutils` and `internal/testutils` are packaged under the same name, `testutils`. To use them separately in the `internal/lvmd/*_test.go` files, we'd have to rename one of them in the import statement. That's a bit of a hassle, so I decided to include `RequireRoot` here instead.